doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -81,7 +81,7 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
   * :kconfig:option:`CONFIG_BT_PER_ADV_SYNC_MAX`
   * :kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE`
   * :kconfig:option:`CONFIG_BT_DEVICE_NAME`
-  * :kconfig:option:`CONFIG_CBKPROXY_OUT_SLOTS` on one core must be equal to :kconfig:option:`CONFIG_CBKPROXY_IN_SLOTS` on the other.
+  * :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_OUT_SLOTS` on one core must be equal to :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_IN_SLOTS` on the other.
```

Comment:
I recently heard over lunch that all Kconfig renames as well as public APIs should follow the deprecation procedure (meaning that we mark them as deprecated and keep as such for 2 releases). But I guess it's up to @doki-nordic and @KAGA164 if this applies to this case as well.